NumberInput/README.md

```diff
@@ -0,0 +1,46 @@
+
+# NumberInput Interface
+
+**Status**: Proposed
+
+| Name        | ID                                                                |
+| :---------- | :---------------------------------------------------------------- |
+| NumberInput | c5a9558b2664aed7dc3e6123436d544f13ffe69ab0e259412f48c6d1c8588401  |
+
+
+## Description
+
+Interface for number input.
+
+## Functions
+
+`int` - prints prompt message to the user and returns singed integer entered by user
```

Comment:
Lets use `select` in all input interfaces.

Reply:
```suggestion
`get` - prints prompt message to the user and returns singed integer entered by user
```